src/components/Navrail/Navrail.module.scss

```
@@ -0,0 +1,22 @@
+.Menu {
+  padding: 8px;
+  margin-top: 16px;
+  margin-left: 8px;
+  margin-right: 8px;
+}
+
+.Header {
+  flex-direction: row;
+  justify-content: space-between;
+  margin: 16px 8px;
+}
+
+.Footer {
+  margin: 16px 8px 32px;
+}
+
+.Gem {
+  padding: 8px;
```

there's some snappiness to the top of the nav bar because the Gem class makes the button slightly larger than the collapse button

to get the Gem button to match the same styling as the collapse button you can do this:
```suggestion
  padding: 6px;
  border-radius: 10px;
  width: fit-content;
  height: fit-content;
}
```

This will make it match the 28px x 28px dimension